app/tasks/maintenance/move_image_urls_task.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class MoveImageUrlsTask < MaintenanceTasks::Task
+    def collection
+      Podcast.all
+    end
+
+    def process(podcast)
+      if podcast.extras["image_urls"].nil?
+        images = []
+
+        podcast.image_urls.each do |image|
+          images << {url: image.url, height: image.height, width: image.width}
+        end
+
+        podcast.extras = {image_urls: images}
+        podcast.save!
```

Comment:
If we add error handling here it will be better in case we can log if a problem exists.

Reply:
Maintask will do it for us if there will be a problem.